packages/slack-handler/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@debugr/slack-handler",
+  "version": "2.0.0",
+  "description": "Elastic log handler to debugr",
+  "author": {
+    "name": "Dan Kadera",
+    "email": "dan.kadera@cdn77.com"
+  },
+  "license": "MIT",
+  "engines": {
+    "node": ">=12.0"
+  },
+  "dependencies": {
+    "@debugr/core": "^2.0",
+    "axios": "^0.27.2"
```

Comment:
nešlo by použít spíš `node-fetch`? Axios je jednak asi 5x větší a jednak bych rád používal věci co nejbližší nativním APIs; `fetch` je nativní API v browseru a `node-fetch` je efektivně jeho polyfill pro node, ale je šance, že jednou v node bude `fetch` nativně, a pak bude stačit odmazat / upravit `import fetch from 'node-fetch'` a bude o závislost míň..

Reply:
done